comment.py

```diff
@@ -14,7 +14,7 @@
     agent = CommentAgent(
         repo_name=REPO_NAME,
         branch_or_prnum=BRANCH_OR_PR_NUMBER,
-        branch=MAIN_BRANCH,
+        main_branch=MAIN_BRANCH,
```

Comment:
Looks like you nicked 'branch' and put 'main_branch' in its stead, but did you check the consequences? Make sure you update the usages as well. Changes propagate my friend, much like the ripples of my endless tears seeing unused variables.